### ISSUE TITLE
no interaction between ai and user

### ISSUE THREAD

**Issue by absarbandayy** [FutureProofHomes/wyoming-enhancements#18]
i cannot hear anything at output although i did all steps well also while checking the outputs on the code i am getting test audio on the same output but AI is not giving audio output

**Comment by FutureProofHomes** [comment]
Hello @absarbandayy.  When you say "AI is not giving audio output" I'm assuming you're saying the voice isn't responding to you.

1. Is your pipeline working if you chat with it though your computer and the chat interface?  https://www.home-assistant.io/voice_control/troubleshooting/
2. If you speak the wake word do you LEDs light up?
3. If the LEDs do light up can you again debug your pipeline and see your speech is being transcribed to text?

Paste any error logs you have to help troubleshoot.  Thnx!

**Comment by absarbandayy** [comment]
yes my pipeline is working when i chat with it .. i am using it on a raspberrypi 5 i have not used 2michat or 4michat ...mic input from a usb sound driver and output through hdmi
